### ISSUE TITLE
lennokit

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#11001]
**Käsitteen tyyppi**

Muutos olemassa olevaan käsitteeseen

**prefabel**

[lennokit](http://www.yso.fi/onto/yso/p8578)

**Tila**

Käsittelyssä

**Ehdotettu muutos**

pitäisikö tämän merkitystä rajata sulkutarkenteella tai mahdollisesti muuttaa hierarkiaa, kun nyt yläkäsitteenä leikkikalut?
Ja vaikka yläkäsitteenä on leikkikalut on ryhminä nyt:
67 Sodankäynti. Sotatekniikka. Maanpuolustus. Aseet
98 Vapaa-aika. Harrastukset. Matkailu. Käsityöt

**Perustelut ehdotukselle**

Käytetty Melindassa droneja käsittelevissä aineistoissa, esimerkiksi:
https://melinda.kansalliskirjasto.fi/byid/018658370
lennokit & miehittämättömät ilma-alukset haku tuottaa Melindassa 130 viitettä

Kielitoimiston sanakirja:
https://www.kielitoimistonsanakirja.fi/#/lennokki
lennokki
(pieni) miehittämätön lentokone (myös harrastus- t. leikkivälineenä).

**Ehdottajan organisaatio**

kk/il

**Comment by mariaforsen** [comment]
Kyllä, pitäisi selkeyttää, sv. on modellflygplan =leikkikalu/harrastus. Ryhmä 67 siis väärä ruotsin kannalta. 

Voisiko dronet/droonit olla erillisenä terminä, vai ohjataanko tämäntyyppiset lennokit edelleen KÄYTÄ [miehittämättömät ilma-alukset](https://finto.fi/yso/fi/page/p24149) ? 

https://www.kielitoimistonsanakirja.fi/#/drone?source=suggestion
https://termipankki.fi/tepa/fi/haku/lennokki
https://www.sanakirja.fi/custom/fi-en-sv/lennokki

**Comment by mirmaid** [comment]
Minkähänlaista jaottelua tässä halutaan? Käytön mukaan (harrastus/sotilaallinen/tutkimus)? 😓

YSAssa tällä on monihierarkia, joka vastaa ryhmiä - YSOssa toinen yläkäsite on jätetty pois.

**Comment by jarmosaarikko** [comment]
Tarvittaessa lennokit vosi olla  ohjausterminä miehittämättömälle ilma-alukselle.   
- olisiko ihan hassua muodostaa tarkenne  toisen ohjaustermin avulla: lennokit (droonit), muuten voisi olla lennokit (ilma-alukset)
- Mutta on täysin mahdollista jättää se tuohon nykyiseenkin käsitteesen, jolloin jätetään ehdotukseksi.

**Lennokit** käsitteestä pitäisi poistaa ryhmä 67, mutta se pitäisi lisätä käsitteelle **miehittämättömät ilma-alukset.**

**Comment by ilona-x** [comment]
Englanniksi ja ruotsiksi termi viittaa mallilennokkeihin, pitäisikö päätermin olla myös suomeksi mallilennokit?
https://fi.wikipedia.org/wiki/Mallilennokki
Myös LCSH-linkki viittaa tähän merkitykseen Airplanes--Models http://id.loc.gov/authorities/subjects/sh85002841

HUOM. lennokkien wikidata-linkki taitaa nyt olla virheellisesti model aircraft building (Q210403), kun oikeampi olisi varmaan
https://www.wikidata.org/wiki/Q60055387

**Comment by ilona-x** [comment]
YSOssa

**miehittämättömät ilma-alukset h**ttp://www.yso.fi/onto/yso/p24149
KT    droonit
KT    miehittämättömät ilmakulkuneuvot
KT    miehittämättömät lentolaitteet

**multikopterit** http://www.yso.fi/onto/yso/p29564


----

Drone aircraft http://id.loc.gov/authorities/subjects/sh85039623
Variants
    Drones (Aircraft)
    Pilotless aircraft
    Remotely piloted aircraft
    UAVs (Unmanned aerial vehicles)
    Unmanned aerial vehicles

**Comment by ilona-x** [comment]
**fi        robottikopteri; robottilennokki; drooni**
en      unmanned aerial vehicle; UAV; unmanned aircraft; drone

määritelmä
kuljettajaton ilma-alus, joka pystyy lentämään joko itsenäisesti automaattiohjauksella tai ihmisen suoraan ohjaamana etäohjauksella

huomautus
Alun perin robottikopterit ovat olleet sotilaskäytössä. Nykyisin niitä on myös siviilikäytössä sekä yksityisillä ihmisillä että yrityksillä. Robottikopterien avulla voi esimerkiksi kuvata ympäristöä (kuvaus- tai kamerakopteri) tai tiettyä kohdetta tai toimittaa paketteja (jakelukopteri).
Droonilla viitataan joskus myös maalla tai vedessä kulkeviin kuljettajattomiin laitteisiin.

Lähde: Tietotekniikan termitalkoot, 2018-08-26 https://termipankki.fi/tepa/fi/haku/drooni

**Comment by ilona-x** [comment]
Pitäisköhän droonit ohjata myös multikopterit-termiin? https://github.com/Finto-ehdotus/YSE/issues/4985

Ja robottikopterit ja robottilennokit voisi myös lisätä ohjaustermeiksi miehittämättömille ilma-aluksille ja multikoptereille? Ja samoin tuon Jarmon ehdottaman lennokit (droonit).

**Comment by tpalonen** [comment]
Ei-sotilaallisten lennokkien osalta tässä on vielä se, että puhutaanko lentävistä vai ei-lentävistä vempaimista. Tämä vaikuttaa ainakin linkityksiin ja kielivastineisiin. Suomen kielessä lennokki ymmärtääkseni viittaa lentävään laitteeseen, ainakin jos Suomi-Wikipediaa ja omaa intuitiotani on uskominen. Kuitenkin esim. enkun termissä model airplane / model aircraft ei niinkään ole olennaista lentääkö laite, vaan termi tarkoittaa lentokoneen pienoismallia, ja se on voitu tehdä joko lentokyvyttömäksi koriste-esineeksi tai sitten sitä voi myös lennättää. Lähipiirissäni on lajin harrastaja ja laitoin hänelle juuri aiheesta kysymyksen, kun en ole itse ihan varma tuosta lennokki-sanasta.

Vrt. vastineiksi Wikipediassa merkatut: https://fi.wikipedia.org/wiki/Mallilennokki ja https://en.wikipedia.org/wiki/Model_aircraft

**Comment by tpalonen** [comment]
Asiantuntijaa on kuultu: lentämättömiä pienoismalleja ei kutsuta lennokeiksi, vaan nimitys tällaisille on pienoismallilentokone. Näitä siis harrastajat rakentavat ja laittavat sitten esim. kotiinsa esille vähän kuin purjelaivojen pienoismallit yms. Näille paras vastine englanniksi lienee tuo model airplane. Eli toisin sanoen lennokki ei ole sama asia kuin model airplane. Lennokkeja ovat sellaiset asiat jotka lentävät. Kannattanee miettiä myös ruotsin termiä vielä tämän pohjalta?

**Comment by mirmaid** [comment]
^ Muun kuin harrastajan on varmaan vaikea hoksata tätä eroa, siis mallilennokit (lentävät) vs. pienoismallilentokoneet.

Pitäisikö multikopterit siirtää miehittämättömien ilma-alusten alle?

robottilennokit KÄ miehittämättömät ilma-alukset (?)
robottikopterit KÄ multikopterit (?)

lennokit (droonit) KÄ miehittämättömät ilma-alukset (?)

droonit (multikopterit) KÄ multikopterit (?)

**Comment by mariaforsen** [comment]
Eli mitä on tarkoitus tehdä nykyiselle lennokit-käsitteelle? Voidaan muuttaa ruotsiksi modellflyplan > drönare, mutta se on sitten tosiaan "drooni". Ja halutaan tietysti myös pitää sanastossa modellflygplan, olisiko sitten lisättävä suomeksi "pienoismallilentokoneet"? (Sanakirjoissa modellflygplan on jostain syystä lennokki, esim. https://www.sanakirja.fi/custom/fi-en-sv/modellflygplan )

**Comment by mirmaid** [comment]
^ Varmaan pitää ratkaista jotenkin Tuomaksen kommentti pienoismallilentokoneista ja mallilennokeista. Jos mallilennokit ovat aina lentäviä pienoismalleja olemassa olevista oikeista lentokoneista, niin on kuitenkin myös vastaavia lennokkeja, jotka eivät jäljittele tarkasti mitään oikeaa konetta. Jossain oli nimitys "taitolennokki". Ja vain hc-harrastaja ymmärtää, että pienoismallilentokone ei lennä, jos mallilennokki on myös pienoismalli oikeasta lentokoneesta.

Lennokkiharrastaja voi ilmeisesti olla, vaikka oma lentohärpäke olisi drooni. _Jos haluat nopeasti ilmaan, niin valmiiden lennokkien ja dronejen tarjonta on laaja, joten varsinainen rakentaminen jää minimiin. -- Kerhon minidronella pääsee lennättämisen makuun jo kerhoillan aikana ja seuraavaksi pääsee harjoittelemaan kerhon opetuslennokilla yhdessä opettajan kanssa._ https://seinajoenlennokkikerho.yhdistysavain.fi/kerho-info/

**Comment by ilona-x** [comment]
eilen tullut muutosehdotus lennokit -käsitteeseen https://github.com/Finto-ehdotus/YSE/issues/11519

**Comment by mirmaid** [comment]
Ja toisaalta uutisista päätellen ero muuttuu vain pienemmäksi. Jos sotilaskäyttö erotettaisiin, jäisi edelleen harrastusdroonien lisäksi tutkimustoiminta, esim. kartoitus droonien avulla.

**Comment by mirmaid** [comment]
Vähän vastaava tapaus aiemmin: https://github.com/Finto-ehdotus/YSE/issues/9391

Jospa lisättäisiin ehdotettuja ohjaustermejä, korjattaisiin kaikki sotilaslennokkeja ja tutkimuskäyttöä koskevat viitteet > miehittämättömät ilma-alukset, ja sen jälkeen pohdintaan, olisi ratkaisu harrastuslennokeille esim. "lennokkiharrastus". Jos kyse on pienoismalleista, on mahdollista kuvailla lentokoneet + pienoismallit. Voi kyllä olla, että tällaiseenkin aineistoon tulisi helposti lisättyä lennokkiharrastus - näin ei-harrastajana on vaikea nähdä että se olisi suuri virhe, määritelmä on toki mahdollista lisätä lennokkiharrastus-termille. Mutta voidaan siis tehdä tuo ehdotettu korjaus ensin.

**Comment by YSO-kokous** [comment]
PÄÄTÖS

Tehdään käyttäjä mirmaidin ehdottamat muutokset eli:
robottilennokit KÄ miehittämättömät ilma-alukset
robottikopterit KÄ multikopterit
lennokit (droonit) KÄ miehittämättömät ilma-alukset
droonit (multikopterit) KÄ multikopterit
Lisäksi korjataan viitteitä eli: sotilaslennokkeja ja tutkimuskäyttöä koskevat viitteet > miehittämättömät ilma-alukset. Jos kyse on pienoismalleista, on mahdollista kuvailla: lentokoneet + pienoismallit.
Lisäksi YSO-ylläpito tutkii, siirretäänkö multikopterit miehittämättömien ilma-alusten alle.

Ehdotus ottaa sanastoon käsite lennokkiharrastus ei saanut kannatusta.

**Comment by mirmaid** [comment]
^ Palataan lennokkiharrastukseen/harrastelennokkeihin korjausten jälkeen.

**Comment by mirmaid** [comment]
Korjattu alusta jonkin matkaa; kun sotilas- ja tutkimuskäyttö (+ siihen rinnastettava, esim. maatalouskäyttö) korjataan pois, viitteisiin jää drooniharrastusta, pienoismalleja ja muuta sekalaista.

**Comment by ilona-x** [comment]
> viitteisiin jää drooniharrastusta, pienoismalleja ja muuta sekalaista.

Eikö drooniharrastuksen yhteydessä voisi käyttää samoja termejä kuin muistakin drooneista eli [multikopterit](https://finto.fi/yso/fi/page/p29564) ja [miehittämättömät ilma-alukset](https://finto.fi/yso/fi/page/p24149)?

**Comment by mirmaid** [comment]
^ Periaatteessa, mutta pitäisikö ne kuitenkin yhdistää ennemmin lennokkiharrastukseen kuin sotilaalliseen ja muuhun ammattikäyttöön.

**Comment by ilona-x** [comment]
Jos esimerkiksi aineisto käsittelee droonikuvausta harrastuksena, olisi mielestäni luontevaa käyttää kuvailussa samaa termiä kuin mitä käytetään aineistoissa, mitkä käsittelevät esim. droonikuvausta tutkimustoiminnan yhteydessä.

En tiedä, onko mahdollista löytää termi, joka kattaisi kaiken harrastustoiminnan erilaisten lennokkien kanssa ja onko nämä kaikki harrastukset järkevä niputtaa yhteen, kun tosiaan lennokkeja on niin monenlaisia, mallilennokeista multikoptereihin. 🤔 https://www.ilmailuliitto.fi/laji/lennokit/lennokkien-tyyppeja/

**Comment by mirmaid** [comment]
^ Toki voi käyttää myös sekä-että, jos saadaan harrastustoiminnalle sopivan kattava termi. Jätin nuo korjaamatta siksikin, kun ei ehkä vielä ole ihan selvillä, millaista lopputulosta tavoitellaan hakijaa ajatellen.

**Comment by mariaforsen** [comment]
Lennokit on edelleen modellflygplan, mitä tälle on nyt tarkoitus tehdä? Modellflygplan kattaa lähinnä pienoismalleja, eikä ruotsiksi löydy sellaista termiä, joka kattaisi sopivasti erilaisia merkityksiä. SAOssa on päätermeinä drönare (ohjausterminä obemannande luftfarkoster) ja modellflygplan. 
https://kaino.kotus.fi/finsk-svensk/?p=qs-article&suru_id=SURU_4dd53f076dae04cebb843ba0dcce71a5&list_id=1&keyword=lennokki&word=lennokki

Modellflygplan siis tämänkaltaisia https://www.aviationshop.fi/lelut-lennokit-c-80.html Voisiko suomeksi olla lennokit (pienoismallit)?

**Comment by mirmaid** [comment]
^ Tarkoitus on keskustella juuri siitä 😅. Ks.

https://github.com/Finto-ehdotus/YSE/issues/11001#issuecomment-1460758329
https://github.com/Finto-ehdotus/YSE/issues/11001#issuecomment-1539633867

Lennokkiharrastajan lennokki ei siis välttämättä ole oikean lentokoneen pienoismalli, vaan se voi olla vain tekijänsä harrastustarpeita palveleva lentovempain. Jotkut rakentavat tarkkoja pienoismalleja, jotka ovat vain esillä ihailtavana, toiset pienoismalleja jotka lentävät, ja kolmannet lennokkeja jotka lentävät eivätkä ole pienoismalleja. Kukaan muu kuin nämä harrastajat tuskin osaa käyttää niistä oikeaoppisesti eri nimityksiä. Huom. Melindaa tutkaillessa löytyi myös termi _liidokit_, joka meillä näköjään on 😅. 

https://melinda.kansalliskirjasto.fi/byid/019019319
https://melinda.kansalliskirjasto.fi/byid/016390400
https://melinda.kansalliskirjasto.fi/byid/012056557
https://melinda.kansalliskirjasto.fi/byid/013789170
https://melinda.kansalliskirjasto.fi/byid/013788870
https://melinda.kansalliskirjasto.fi/byid/013789239

**Comment by mirmaid** [comment]
Periaatteessa voisi vaikka tehdä > _lennokit ja liidokit_ KT mallilennokit. Vaikka Tuomaksen harrastajatutun mukaan lentämättömiä pienoismalleja ei kutsuta lennokeiksi, näyttää Melindassa käytetyn tuota lennokit-termiä myös näistä (jos nyt osaan oikein tulkita pakkausten kuvia). _Lentokoneet + pienoismallit_ toki riittäisi näihin ja on käytettykin lennokit-termin lisäksi.

**Comment by mariaforsen** [comment]
Mitä kaikkea lennokit ja liidokit kattaisi? Sekä pienoismalleja että lentäviä lennokeita?
https://kaino.kotus.fi/finsk-svensk/?p=qs-article&suru_id=SURU_d65bc852de199a97d3600c1698a86b03&list_id=1&keyword=liidokki&word=liidokki

**Comment by mirmaid** [comment]
^ No näin ajattelin. Siis kun muut kuin hc-harrastajat tuskin osaavat tehdä tuota eroa lentävien ja lentämättömien välillä mitenkään, jollei se suorastaan lue termissä (koska pienoismalli voi käsittääkseni lentää tai olla vain koristeena, ja lennokki joka lentää ei välttämättä ole pienoismalli, mutta voi olla). Ja jos halutaan kertoa että kyseessä on pienoismalli, niin käytettävissä on termi pienoismallit ja lisäksi lennokit ja/tai lentokoneet.

**Comment by YSO-kokous** [comment]
PÄÄTÖS:

Yhdistetään lennokit ja liidokit yhdeksi käsitteeksi: lennokit ja liidokit. Tätä voi käyttää kuvailussa erityisesti kaikista lentävistä asioista. Tätä voi käyttää myös asioista, jotka ovat pienoismallityyppisiä. Tätä voi käyttää myös ei-lentävistä asioista. Jos on kyse pienoismallista, kuvailussa voi käyttää myös pienoismallit-käsitettä (ja lentokoneet -käsitteitä).

**Comment by mariaforsen** [comment]
Sv. modellflyg (objekt) HT modellflygplan, modellglidflygplan, flygmodeller, modellflygplan och segelmodeller

https://kaino.kotus.fi/finsk-svensk/?p=qs-article&suru_id=SURU_d65bc852de199a97d3600c1698a86b03&list_id=1&keyword=liidokki&word=liidokki
http://www.modellsegelflyg.se/
https://sv.wikipedia.org/wiki/Modellflyg
https://termipankki.fi/tepa/fi/haku/lennokki
https://flygsport.se/organisation/distrikt/skanes-flygsportforbund/vara-flygsporter/modellflyg
https://www.modellflygforbund.se/grenar/modellflyg/SMFF/om-smff
https://www.regeringen.se/contentassets/926fda31ccf5453e9e5364702c8c5c8c/svenska-rc-flygforbundet_.pdf
https://www.sanakirja.fi/custom/fi-en-sv/liidokki
https://www.kielitoimistonsanakirja.fi/#/lennokki?searchMode=all
https://www.kielitoimistonsanakirja.fi/#/liidokki?searchMode=all
https://www.sanakirja.fi/custom/fi-en-sv/glidflygplan
https://fi.wikipedia.org/wiki/Lennokki 
https://www.biltema.fi/sv-fi/verktyg/utforsaljning-verktyg/glidflygplan-jumbo-2000060377?srsltid=AfmBOooXIVLoMaL_m5RAdm4U09yR-0Mn402Chb6ruZk7_vWI6Bmbq0YU
https://hobbymodeller.se/modellflygplan-byggsatser-och-tillbehor/balsabyggsatser-flygplansmodeller/
https://www.kmfk.org/wordpress/klubben-3/modellflyg/
